### ISSUE TITLE
Typo in documentation

### ISSUE THREAD

**Issue by ghostchibi** [tmedwards/sugarcube-2#27]
https://www.motoslave.net/sugarcube/2/docs/#guide-harlowe-to-sugarcube-specific-macros
Under "Link and Click Macros" the following example code is closed incorrectly:

`<<linkreplace 'Hey there.'>>Hay is for horses.<</link>>`

Not the biggest deal in the world since the actual section on `<<linkreplace>>` has it correctly, but I did wonder for a few minutes why `<</link>>` wasn't closing `<<linkreplace>>` when I did a Ctrl-F to find an example use of the macro.

**Comment by tmedwards** [comment]
Thanks.

**Comment by tmedwards** [comment]
As an FYI for the future.  If I address an issue with a commit, then the issue will automatically be closed once that commit is merged into the release branch—meaning that you don't have to manually close it yourself.

Anyway.  Thanks again for the catch.
